contrib/upload_docs.sh

```diff
@@ -15,6 +43,10 @@ git remote show origin &>/dev/null || git remote add origin https://github.com/o
 git fetch --all
 git checkout -t origin/master -f
 git pull
+if [ "X$with_ucg" = "Xyes" ]
+then
+	git submodule update --init --recursive
+fi
```

Comment:
why is this needed? we already have it in autogen
a developer can't even build docs w/o running autogen first

Reply:
@yosefe not sure where this came from... all I did was squash. I'll remove (both).